packages/components/grid/src/grid.scss

```diff
@@ -208,6 +229,20 @@ td[part~='text-field'] {
   font-weight: 600;
 }
 
+:host([clickable-row]) {
+  [part~='row'] {
+    cursor: pointer;
+
+    &:hover {
```

Comment:
So when i click a row, there is no `:active` styling (unlike in a button for example). Shouldn't we add that here as well?